TestProjects/UniversalGraphicsTest_Foundation/Assets/ControlRenderScale.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+
+public class ControlRenderScale : MonoBehaviour
+{
+    public float playRenderScale = 0.25f;
+    float originalScale = 1.0f;
+
+    // Update is called once per frame
+    void Update()
+    {
+        // this doesn't seem to do anything...
+        // ScalableBufferManager.ResizeBuffers(0.25f, 0.25f);
+    }
+
+    private void OnEnable()
+    {
+        // UniversalRenderPipelineAsset asset = new UniversalRenderPipelineAsset();
```

Comment:
Can this be removed?

Reply:
yup!  Removed

Reply:
Also moved this file into the test directory, no need to be in the outermost directory